### PR TITLE
WIP: Add JWT signing and authentication backend

### PR DESCRIPTION
**Pull request by dnegma**
Add JWS auth backend, handles tokens passed in request cookies or the Authorization header.
Format JWT data to account entity data.
Separate JWT auth into separate namespace.

Added some specs, maybe it makes sense?

### CODE REVIEW

File: src/customs/access.clj [hunk 1]
```diff
@@ -76,11 +78,45 @@
    :body    body
    :headers {"Content-Type" "text/html; charset=utf-8"}})
```
Comment (allenjprice) [inline src/customs/access.clj:88]: This is an embarassingly nit-picky comment, but here goes:

Sometime in between when @jalehman initially wrote this code and when our team expanded to include multiple engineers, we adopted [Readable Clojure](http://tonsky.me/blog/readable-clojure/) as our team's unofficial style guide.

One of the principles in Readable Clojure calls for [two lines of space between top-level forms](http://tonsky.me/blog/readable-clojure/#use-two-empty-lines-between-top-level-forms). The only exception we make to this rule is for `spec` - we only use one line of whitespace between a top-level and its corresponding spec definition.

It looks like the original commit didn't follow this rule, so it's totally reasonable that you would emulate the style found elsewhere in the file!

Would you mind bringing this file (and any others you see violating the "two line" rule) into compliance?

**Review by allenjprice — CHANGES_REQUESTED** [review]
Just one tidbit of feedback here. Otherwise, looks good!

File: src/customs/access.clj (hunk 1 above)
Comment (dnegma) [reply src/customs/access.clj:88]: Will do!

**Review by allenjprice — APPROVED** [review]

**Comment by allenjprice** [comment]
🎉 fantastic work, @dnegma

**Pull request merged**